tei/la-cisma-de-ingalaterra.xml

```diff
@@ -2764,8 +2771,8 @@
             <l>que aquestas cosas no son</l>
             <l>para tratadas así.</l>
             <l>El cielo te guarde. Adiós.</l>
-            <l>Hace que se va.</l>
           </sp>
+          <stage>Hace que se va.</stage>
```

Comment:
I'd just like to point out how interesting this stage direction is. It is a pretend exit. It appears again later, when Ana, who now suffers it, uses it on the king.